### ISSUE TITLE
 There is a question about ground truth

### ISSUE THREAD

**Issue by chenyuanpengcyp** [makedede/MEFNet#1]
During training, do we need additional calibrated HDR image as gt ？

**Comment by zduanmu** [comment]
> During training, do we need additional calibrated HDR image as gt ？

No. We only need a stack of LDR images taken under different exposure values in the training stage.
